### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.BadRequestException
  id: BadRequestException
  children:
  - System.IdentityModel.BadRequestException.#ctor
  - System.IdentityModel.BadRequestException.#ctor(System.String)
  - System.IdentityModel.BadRequestException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.BadRequestException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: BadRequestException
  nameWithType: BadRequestException
  fullName: System.IdentityModel.BadRequestException
  type: Class
  summary: "De uitzondering die wordt gegenereerd wanneer een tokenaanvraag (eerste) niet wordt begrepen door de beveiligingstokenservice (STS)."
  remarks: "De eerste wordt vertegenwoordigd door een <xref:System.IdentityModel.Protocols.WSTrust.RequestSecurityToken>object.</xref:System.IdentityModel.Protocols.WSTrust.RequestSecurityToken>"
  syntax:
    content: 'public class BadRequestException : System.IdentityModel.RequestException'
  inheritance:
  - System.Exception
  - System.IdentityModel.RequestException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.BadRequestException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.BadRequestException
  langs:
  - csharp
  name: BadRequestException()
  nameWithType: BadRequestException.BadRequestException()
  fullName: System.IdentityModel.BadRequestException.BadRequestException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.BadRequestException&quot;> </xref> klasse."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message%2A>eigenschap van het nieuwe exemplaar voor een systeem geleverd bericht met een beschrijving van de fout.</xref:System.Exception.Message%2A> Dit bericht wordt rekening gehouden met de huidige systeemcultuur.       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De gelokaliseerde fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public BadRequestException ();
    parameters: []
  overload: System.IdentityModel.BadRequestException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.BadRequestException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.BadRequestException
  langs:
  - csharp
  name: BadRequestException(String)
  nameWithType: BadRequestException.BadRequestException(String)
  fullName: System.IdentityModel.BadRequestException.BadRequestException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.BadRequestException&quot;> </xref> klasse met een opgegeven foutbericht."
  remarks: "Deze constructor initialiseert de <xref:System.Exception.Message%2A>eigenschap van de nieuwe uitzondering via de `message` parameter.</xref:System.Exception.Message%2A> De inhoud van `message` is bedoeld om te worden ge√Ønterpreteerd door mensen. De aanroeper van deze constructor is vereist om ervoor te zorgen dat deze tekenreeks voor de huidige systeemcultuur gelokaliseerd.       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public BadRequestException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
  overload: System.IdentityModel.BadRequestException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.BadRequestException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.BadRequestException
  langs:
  - csharp
  name: BadRequestException(SerializationInfo,StreamingContext)
  nameWithType: BadRequestException.BadRequestException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.BadRequestException.BadRequestException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.BadRequestException&quot;> </xref> klasse met de geserialiseerde gegevens."
  remarks: "Deze constructor is aangeroepen tijdens de deserialisatie op het object exception verzonden via een stroom opnieuw samenstellen."
  syntax:
    content: protected BadRequestException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Een <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> object dat de gegevens geserialiseerd object bevat."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Een <xref:System.Runtime.Serialization.StreamingContext>object dat de contextuele informatie over de bron- of doelserver bevat.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.BadRequestException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.BadRequestException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.BadRequestException
  langs:
  - csharp
  name: BadRequestException(String,Exception)
  nameWithType: BadRequestException.BadRequestException(String,Exception)
  fullName: System.IdentityModel.BadRequestException.BadRequestException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Initialiseert een nieuw exemplaar van de <xref href=&quot;System.IdentityModel.BadRequestException&quot;> </xref> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering."
  remarks: "Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering moet een verwijzing bevatten naar de vorige uitzondering in de <xref:System.Exception.InnerException%2A>eigenschap.</xref:System.Exception.InnerException%2A> De <xref:System.Exception.InnerException%2A>eigenschap dezelfde waarde die is doorgegeven aan de constructor.</xref:System.Exception.InnerException%2A>       De volgende tabel toont de eerste waarden voor het nieuwe exemplaar.      | Eigenschap | Waarde |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| De verwijzing naar de interne uitzondering. |   | <xref:System.Exception.Message%2A>| De fouttekenreeks bericht. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public BadRequestException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Het foutbericht met uitleg over de reden voor de uitzondering."
    - id: innerException
      type: System.Exception
      description: "De <xref:System.Exception>die de oorzaak van de huidige uitzondering.</xref:System.Exception> Als de `innerException` parameter is niet <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, de huidige uitzondering is opgetreden tijdens een <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blok die verantwoordelijk is voor de interne uitzondering."
  overload: System.IdentityModel.BadRequestException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.IdentityModel.RequestException
  isExternal: false
  name: System.IdentityModel.RequestException
- uid: System.IdentityModel.BadRequestException.#ctor
  parent: System.IdentityModel.BadRequestException
  isExternal: false
  name: BadRequestException()
  nameWithType: BadRequestException.BadRequestException()
  fullName: System.IdentityModel.BadRequestException.BadRequestException()
- uid: System.IdentityModel.BadRequestException.#ctor(System.String)
  parent: System.IdentityModel.BadRequestException
  isExternal: false
  name: BadRequestException(String)
  nameWithType: BadRequestException.BadRequestException(String)
  fullName: System.IdentityModel.BadRequestException.BadRequestException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.BadRequestException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.BadRequestException
  isExternal: false
  name: BadRequestException(SerializationInfo,StreamingContext)
  nameWithType: BadRequestException.BadRequestException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.BadRequestException.BadRequestException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.BadRequestException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.BadRequestException
  isExternal: false
  name: BadRequestException(String,Exception)
  nameWithType: BadRequestException.BadRequestException(String,Exception)
  fullName: System.IdentityModel.BadRequestException.BadRequestException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.IdentityModel.BadRequestException.#ctor*
  parent: System.IdentityModel.BadRequestException
  isExternal: false
  name: BadRequestException
  nameWithType: BadRequestException.BadRequestException
